apps/passport-client/components/shared/PCDCard.tsx

```diff
@@ -0,0 +1,180 @@
+import { PCD } from "@pcd/pcd-types";
+import * as React from "react";
+import { useCallback, useContext, useMemo } from "react";
+import styled from "styled-components";
+import { DispatchContext } from "../../src/dispatch";
+import { usePackage } from "../../src/usePackage";
+import { Button, H4, Spacer, TextCenter } from "../core";
+import { ZuzaluCardBody } from "./ZuzaluCard";
+
+/**
+ * Shows a card in the Passport wallet. If expanded, the full card, otherwise
+ * just the top of the card to allow stacking.
+ */
+export function PCDCard({
+  isZuzaluIdentity,
+  pcd,
+  expanded,
+  onClick,
+  hideRemoveButton,
+}: {
+  pcd: PCD;
+  expanded?: boolean;
+  isZuzaluIdentity?: boolean;
+  onClick?: () => void;
+  hideRemoveButton?: boolean;
+}) {
+  const pcdPackage = usePackage(pcd);
+  const displayOptions = useMemo(() => {
+    if (pcdPackage?.getDisplayOptions) {
+      return pcdPackage?.getDisplayOptions(pcd);
+    }
+  }, [pcd, pcdPackage]);
+  const header = isZuzaluIdentity
+    ? "VERIFIED ZUZALU PASSPORT"
+    : displayOptions?.header ?? "PCD";
```

Comment:
if you separated out `PCDCard` from `ZuzaluCard`, do you still need the special case?

& for typographic consistency, the header should be `toUpperCase()`

Reply:
Yes. Under the hood, a zuzalu card is a semaphore identity pcd, so I am special casing it to make it behave differently from other semaphore identity cards.